ext/standard/tests/streams/gh11418.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+GH-10406: fgets on a redis socket connection fails on PHP 8.3
```

Comment:
@bukka nit: it's GH-11418 not GH-10406 (the same in the commit message, 10406 is the original _feof() behavior change_ issue)

Reply:
I think he really meant it as fixing 10406 for good and, consequentially, fixing 11418 implicitly.